<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EEGAppMenu</class>
 <widget class="QMainWindow" name="EEGAppMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>251</width>
    <height>365</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>16</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>24</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>EEG Applications</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnApplications">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>210</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../../assets/ImageResources.qrc">
      <normaloff>:/ButtonIcons/btnApplications.png</normaloff>:/ButtonIcons/btnApplications.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>210</width>
      <height>80</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCalibration">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>210</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../../assets/ImageResources.qrc">
      <normaloff>:/ButtonIcons/btnCalibration.png</normaloff>:/ButtonIcons/btnCalibration.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>210</width>
      <height>80</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSettings">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>210</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../../assets/ImageResources.qrc">
      <normaloff>:/ButtonIcons/btnSettings.png</normaloff>:/ButtonIcons/btnSettings.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>210</width>
      <height>80</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>251</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../../assets/ImageResources.qrc"/>
 </resources>
 <connections/>
</ui>
